retailcrm/lib/RetailcrmReferences.php

```diff
@@ -340,6 +343,48 @@ public function getApiPaymentTypes()
         return $crmPaymentTypes;
     }
 
+    public function getSite()
+    {
+        try {
+            $response = $this->api->credentials();
```

Comment:
request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function credentials() on null" at /Users/opheugene/sites/prestashop.loc/www2/modules/retailcrm/lib/RetailcrmReferences.php line 349 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function credentials() on null at /Users/opheugene/sites/prestashop.loc/www2/modules/retailcrm/lib/RetailcrmReferences.php:349)"}